### PR TITLE
Geirst/misc cleanup after new document api protocol

### PR DESCRIPTION
**Pull request by geirst**
@toregge and @vekterli please review

### CODE REVIEW

File: vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/BucketSpaceEnumerator.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import java.util.stream.Collectors;
 
 /**
- * TODO description
+ * Class the based on BucketspacesConfig builds a mapping from document type to which bucket space it belongs to.
```
Comment (vekterli) [inline vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/BucketSpaceEnumerator.java]: Typo: "the" -> "that/which" (presumably)

**Review by vekterli — APPROVED** [review]
:+1: minor comment typo, otherwise LGTM

**Comment by geirst** [comment]
@vekterli fixed now

**Review by toregge — APPROVED** [review]
LGTM

**Pull request merged**